core/src/main/java/org/apache/accumulo/core/file/FileOperations.java

```diff
@@ -301,6 +302,16 @@ public RateLimiter getRateLimiter() {
       return this;
     }
 
+    public boolean isAccumuloStartEnabled() {
+      return enableAccumuloStart;
+    }
+
+    @Override
+    public OpenWriterOperation setAccumuloStartEnabled(boolean enableAccumuloStart) {
```

Comment:
The one thing I miss from Review Board is its per review issues.  Those were nice to help track what needed to be done on a review with lots of comments.